MobileApp/babel.config.js

```diff
@@ -2,5 +2,12 @@ module.exports = function (api) {
   api.cache(true);
   return {
     presets: ["babel-preset-expo"],
+    plugins: [
+      ["module:react-native-dotenv", {
```

Comment:
This will be good for test environment and development but not good for the live version. The environment variables will be accessible on the client side bundle.
So we will need to build a proxy and provide authentication for it using user pools.

I will approve now for Demo 3.

Reply:
Agreed